src/plugins/data/public/index.ts

```diff
@@ -109,6 +112,7 @@ export * from './index_patterns';
 export * from './search';
 export * from './query';
 export * from './ui';
+export * from './field_formats';
```

Comment:
Please avoid this (I just removed this line) and use an explicit export for `FormatFactory`, renaming it to `FieldFormatFactory` (consistent prefix), preferably next to the `from '../common/field_formats'` block

Reply:
i need the `FieldFormatsStart` type